schug/endpoints/exons.py

```diff
@@ -27,7 +28,17 @@ def read_exons(
 async def ensembl_exons(build: Build):
     """A proxy to the Ensembl Biomart that retrieves exons in a specific genome build."""
 
-    ensembl_client: EnsemblBiomartClient = fetch_ensembl_exons(build=build)
-    url: str = ensembl_client.build_url(xml=ensembl_client.xml)
+    async def chromosome_stream():
+        for chrom in CHROMOSOMES:
```

Comment:
The actual workaround is downloading exons chromosome by chromosome, which seems to work fine when tested manually using the Ensembl Biomart website